.github/workflows/gitstream.yaml

```diff
@@ -0,0 +1,51 @@
+# Code generated by gitStream GitHub app - DO NOT EDIT
+
+name: gitStream workflow automation
+run-name: |
+  /:\ gitStream: PR #${{ fromJSON(fromJSON(github.event.inputs.client_payload)).pullRequestNumber }} from ${{ github.event.inputs.full_repository }}
+
+permissions: {}
+
+on:
+  workflow_dispatch:
+    inputs:
+      client_payload:
+        description: The Client payload
+        required: true
+      full_repository:
+        description: the repository name include the owner in `owner/repo_name` format
+        required: true
+      head_ref:
+        description: the head sha
+        required: true
+      base_ref:
+        description: the base ref
+        required: true
+      installation_id:
+        description: the installation id
+        required: false
+      resolver_url:
+        description: the resolver url to pass results to
+        required: true
+      resolver_token:
+        description: Optional resolver token for resolver service
+        required: false
+        default: ''
+
+jobs:
+  gitStream:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    name: gitStream workflow automation
```

Comment:
Won't we need explicit `write` permissions here to allow for the action to write comments and/or change labels? 🤔 
https://github.com/mui/mui-x/blob/master/.github/workflows/no-response.yml#L19

Reply:
good question ... I think the gitStream application that needs to be hooked to the org does have that permission (once approved and hooked)

Reply:
Why does it need to be installed on the GitHub organization? I had understood that gitstream can run directly in GitHub actions, where it has the required permission.

Reply:
Actually, I'm confused is this open-source? https://linearb.io/pricing suggests we have to pay $50k/year which is not going to work.

Reply:
Installed

Reply:
@BenLloydPearson thanks for the clarification.